src/main copy.ts

```diff
@@ -0,0 +1,18 @@
+import Vue from "vue";
+import App from "./App.vue";
```

Comment:
Why is this part of the source tree ? @jkamoona you need to be more careful about what files you're committing